community.json

```diff
@@ -1730,6 +1730,13 @@
         "revision": "795449cb43ccdba949c814746d6ea90eee0a7259",
         "state": "working",
         "url": "https://github.com/aymhce/yunohost_ynh"
+    },
+     "zabbix": {
+        "branch": "master",
+        "level": 2,
+        "revision": "9386c5b2",
```

Comment:
I think you need to put the full commit id here, not just a short one ?

Reply:
Gitlab works with small ids like this : 
https://framagit.org/Mickael-Martin/zabbix_ynh/commits/master